system/HTTP/Response.php

```diff
@@ -467,13 +467,8 @@ public function getJSON()
 
 		if ($this->bodyFormat !== 'json')
 		{
-			/**
-			 * @var Format $config
-			 */
-			$config    = config(Format::class);
-			$formatter = $config->getFormatter('application/json');
-
-			$body = $formatter->format($body);
+			$format = Services::format();
+			$body = $format->getFormatter('application/json')->format($body);
```

Comment:
Maybe we could do it in one line? It's not that we use this class later. In other methods too.
```suggestion
			$body = Services::format()->getFormatter('application/json')->format($body);
```